publish/aliPublishS3

```diff
@@ -769,6 +771,24 @@ def sync(pub, architectures, s3Client, bucket, baseUrl, basePrefix, rules,
         error("%s / %s / %s: publish script failed with %d",
               arch, pack["name"], pack["ver"], rv)
 
+  # Update the alienv script in CVMFS if it has changed
+  try:
+    new_alienv = dirname(realpath(__file__)) / ".." / "cvmfs" / "alienv"
+    old_alienv = "/cvmfs/alice.cern.ch/bin/alienv"
+    
+    # Create backup with timestamp if files are different
+    if not filecmp.cmp(new_alienv, old_alienv, shallow=False):
+        timestamp = datetime.now().strftime("%Y%m%d-%H%M%S")
+        backup_alienv = f"/cvmfs/alice.cern.ch/bin/alienv.safe-{timestamp}"
```

Comment:
Maybe we should have a separate folder for the backups (or limit the backups to just a few).